styles/vertical-timeline.scss

```diff
@@ -1,41 +1,123 @@
+$timeline-indent-gap: 25px;
+$timeline-line-width: 10px;
+$timeline-widget-inset: 15px;
+$point-event-widget-width: 25px;
+$range-event-widget-width: 33px;
+$event-widget-border-width: 4px;
+
 /* The actual timeline (the vertical ruler) */
 .timeline {
-    position: relative;
-    max-width: 1200px;
-    margin: 0 auto;
+  position: relative;
+  max-width: 1200px;
+  margin: 0 auto;
 
-    &::after {
-    content: '';
-    position: absolute;
-    width: 10px;
-    background-color: var(--background-secondary);
-    top: 0;
-    bottom: 0;
-    left: 50%;
-    margin-left: -3px;
-    }
+  &::after {
+  content: '';
+  position: absolute;
+  width: $timeline-line-width;
+  background-color: var(--background-secondary);
+  top: 0;
+  bottom: 0;
+  left: calc(50% - $timeline-line-width / 2);
+  }
 }
 
 /* Container around content */
 .timeline-container {
-    padding: 10px 40px;
-    position: relative;
-    background-color: inherit;
-    width: 50%;
+  padding: 10px 40px;
+  position: relative;
+  background-color: inherit;
+  width: calc(50% - (var(--timeline-indent) * $timeline-indent-gap));
+
+  /* Arrow pointing from note to widget */
+  &:not(.timeline-tail){
```

Comment:
I actually explicitly stopped the arrow from rendering on the endnote by adding `:not(.timeline-tail)`